pages/common/xml-unescape.md

```diff
@@ -0,0 +1,16 @@
+# xml-unescape
+
+> Un-escape special XML characters.
```

Comment:
No hyphen is needed here.
```suggestion
> Unescape special XML characters.
```

Reply:
xml-unescape.md:> Unescape special XML characters.
xml-unescape.md:- Unescape special XML characters from a string:
xml-unescape.md:- Unescape special XML characters from stdin: